src/core/exceptions.c

```diff
@@ -560,9 +560,9 @@ MVMObject * MVM_exception_backtrace(MVMThreadContext *tc, MVMObject *ex_obj) {
             MVMBytecodeAnnotation *annot = MVM_bytecode_resolve_annotation(tc, &cur_frame->static_info->body,
                                                 offset > 0 ? offset - 1 : 0);
             MVMuint32             fshi   = annot ? (MVMint32)annot->filename_string_heap_index : -1;
-            char            *line_number = MVM_malloc(16);
+            char         line_number[11] = {0};
```

Comment:
Would be nice to have a comment on why 11 is the right number. I guess it's because you need a maximum of 10 decimal digits + the NULL character to represent a 32 bit line number. Having that documented makes it clear that it needs adjusting if we ever move to 64 bit line numbers.